tensorpipe/test/channel/basic/basic_test.cc

```diff
@@ -11,7 +11,7 @@
 
 namespace {
 
-class BasicChannelTestHelper : public ChannelTestHelper {
+class BasicChannelTestHelper : public ChannelTestHelper<tensorpipe::CpuTensor> {
```

Comment:
Wondering if we should introduce some `CpuChannelTestHelper` shortcuts?

Reply:
Will do.